lib/src/payments/p2ms.dart

```diff
@@ -0,0 +1,217 @@
+import '../utils/constants/op.dart';
+import 'package:meta/meta.dart';
+import '../utils/script.dart' as bscript;
+import '../models/networks.dart';
+import 'dart:typed_data';
+import 'package:bip32/src/utils/ecurve.dart' show isPoint;
+
+class P2MS {
+  P2MSData data;
+  NetworkType network;
+  List<dynamic> _chunks;
+  Map _temp;
```

Comment:
_temp does not make sense

Reply:
Do you mean that _temp should not exist and that I should change the value of P2MSData data directly? 
The following code:
`_temp['m'] = _chunks[0] - OPS['OP_RESERVED'];
` 
should be: 
`data.m = _chunks[0] - OPS['OP_RESERVED'];
` instead?

Reply:
Yes, you're right 👍

Reply:
I removed all _temp usage, but it has caused the code to regress and fail 3 test. I reference this regression in this [commit](https://github.com/anicdh/bitcoin_flutter/pull/1/commits/014182049e91efab8145872b2def4b757ffc0308). I a unsure of how to fix this. I don't think we need this test anymore, but it feels wrong to delete test, in response to code regression.